spec/PlatformJson.spec.js

```diff
@@ -123,7 +123,7 @@ describe('PlatformJson class', function () {
 describe('ModuleMetadata class', function () {
     it('Test 010 : should be constructable', function () {
         var meta;
-        expect(function name (params) {
+        expect(function () {

```

Comment:
This one looked like a snippet expansion where the placeholders weren't replaced.